hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -186,7 +186,8 @@ public static void setCarbonReadSupport(Class<? extends CarbonReadSupport> readS
   }
 
   public static CarbonTablePath getTablePath(Configuration configuration) throws IOException {
-    AbsoluteTableIdentifier absIdentifier = getAbsoluteTableIdentifier(configuration);
+    AbsoluteTableIdentifier absIdentifier =
+        getCarbonTable(configuration).getAbsoluteTableIdentifier();
```

Comment:
You can modify `getAbsoluteTableIdentifier` directly instead of modifying all callers